## Analytical Techniques{#anal}

Figure @fig:intro-metabolomes introduces the concept of *metabolome*\footnotemark{}.
\footnotetext{This concept will be discussed later.}
It will be used to guide the reading, detailing the main techniques linked to each *metabolome* level.

![**Schematic view of the *metabolomes*.**](images/intro-metabolomes.pdf "intro-metabolomes"){#fig:intro-metabolomes short-caption="Schematic view of the *metabolomes*" align="center" width="100%"}

The *plant metabolome* represents the first level.
It is an abstract notion and is actually not accessible.
The word *metabolome* was introduced in 1998 [@doi:10/fkzwzj].
As a first appproximation, the metabolome corresponds the small molecules, often termed metabolites, that can be found in an organism.

The second level is the *extracted metabolome*.
It represents all metabolites that chemists are able to extract from the plant.
While not everything is extracted, it is to be noted that some additional chemicals, not originally present in the plant, are also produced during this step.

The third level corresponds to the *detected metabolome*.
After extraction, only a part of what has been extracted can be detected by modern instruments.
This part varies depending on the technology(ies) used for the analyses.

Furthermore, only a portion of the *detected metabolome* can be *qualified*, and an even smaller portion *quantified* with sufficient confidence.

### The *Plant Metabolome*

To refine the first approximation of *plant metabolome* it is first necessary to better define the source it putatively originates from, the plant.

`//TODO`{.red}.

In reality, the small can originate from other sources, such as endophytes present inside the plant.

microbiome effect on metabolome [@doi:10.1073/pnas.0812874106]

reading the metabolic fine print [@doi:10.1038/embor.2008.236]

in reality, holobiont [@doi:10.1016/j.mib.2017.07.001]

if needed hologenome [@doi:10.1186/s40168-018-0457-9]

big endophyte book [@doi:10.1007/978-3-319-90484-9_9]

endophytes [@doi:10.3389/fmicb.2016.01538]

number 2 [@doi:10.3389/fbioe.2020.00467]

Targeted approaches, because it only consider only a restricted set of metabolites, rely on the assumption that the knowledge about metabolites and the respective pathways in which they are involved is complete.
Untargeted approaches, in contrary, attempt to analyze all metabolites.
They are thus capable to capture information from unknown metabolites, or not known to be produced by a specific organism under a specific condition.
The advantage of a targeted approach is that it provides more precise and accurate quantitative information.
Thus, untargeted approaches cannot completely replace targeted approaches, but offer additional information that is lost by targeted approaches.
The classical reductionist approaches used in analytical chemistry start from the least targeted approach possible to finally lead to highly targeted analyses.

In this process, the first reduction consists in selecting samples that are (ideally) representative of a given organism, at a given time, under a given condition.
What is often called the *metabolome* of an organism is therefore in reality more a *mixture of some metabolomes*.

The second reduction is the method of extraction used.

`//TODO`{.red}.

Explain need to limit dirty biological matrices

### The *Extracted Metabolome*

`//TODO`{.red}.

#### Extraction

`//TODO`{.red}.

#### Established Extracts

`//TODO`{.red}.

#### Chromatography

GC more theoretical plates

UPLC [@doi:10.1021/ac701807v]

`//TODO`{.red}.

### The *Detected Metabolome*

Only parts detected [@doi:10.1002/jssc.200900152]

number 2 [@doi:10.1111/j.1755-0998.2010.02854.x]


`//TODO`{.red}.

#### Photo Diode Array

`//TODO`{.red}.

#### Mass Spectrometry

GC reproducible

ms-metabolomics [@doi:10.1074/jbc.R111.238691]

number 2 [@doi:10.1002/cbic.200500151]

multiple ionization [@doi:10.1021/ac701982e]

`//TODO`{.red}.

The Orbitrap: A new mass spectrometer [@doi:10.1002/jms.856]

##### Tandem Mass Spectrometry

`//TODO`{.red}.

#### Evaporative Light Scattering Detector

`//TODO`{.red}.

Universal quantification of structurally diverse natural products using an evaporative light scattering detector [@doi:10.1021/np300034c]

Development and validation of an analytical method based on HPLC- ELSD for the simultaneous determination of rosmarinic acid, carnosol, carnosic acid, oleanolic acid and ursolic acid in rosemary [@doi:10.3390/molecules24020323 ]

#### Charged Aerosol Detector

`//TODO`{.red}.

Application of high-performance liquid chromatography with charged aerosol detection (LC–CAD) for unified quantification of synthetic cannabinoids in herbal blends and comparison with quantitative NMR results [@doi:10.1007/s11419-017-0392-7]

Seeking universal detectors for analytical characterizations. [@doi:10.1016/j.jpba.2018.09.029]

Universal response model for a corona charged aerosol detector [@doi:10.1016/j.chroma.2010.09.056]

Comparison of UV and charged aerosol detection approach in pharmaceutical analysis of statins [@doi:10.1016/j.talanta.2008.12.057]

Comparison between evaporative light scattering detection and charged aerosol detection for the analysis of saikosaponins [@doi:10.1016/j.chroma.2010.04.047]

Comparison of the response of four aerosol detectors used with ultra high pressure liquid chromatograph [@doi:10.1016/j.chroma.2011.01.062]

#### Nuclear Magnetic Resonance & Other Spectroscopic Methods

NMR metabolomics [@doi:10/dgtr9s]

Two dimensional NMR spectroscopic approaches for exploring plant metabolome: A review [@doi:10.1016/j.jare.2014.10.003]

Tutorial for the structure elucidation of small mole- cules by means of the LSD software [@doi:10.1002/mrc.4612]

NMR of natural products at the ‘nanomole-scale’ [@doi:10.1039/b920545b]

Computer-aided Dereplication and Structure Elucidation of Natural Products at the University of Reims [@doi:10.1002/minf.201700027]

`//TODO`{.red}.

### The *Qualified Metabolome*

`//TODO`{.red}.

#### Structural Annotation

`//TODO`{.red}.

##### Spectral Similarity

`//TODO`{.red}.

Dot product [@doi:10/bhw492]

Weightings [@doi:10.1093/bioinformatics/bts083]

New methods (Spec2Vec and consorts)

#### Dereplication

`//TODO`{.red}.

#### *De novo* Annotation

`//TODO`{.red}.

#### Structural Elucidation

`//TODO`{.red}.

combination nmr ms [@doi:10.1002/anie.200603821]

### The *Quantified Metabolome*

only targeted [@doi:10.2174/138920111795909096]

`//TODO`{.red}.