## Computational Era {#sec:computational}

Switch to the macroscope

Cybernetics/Systemics

`//TODO`{.red}.
NP Rollinger
Virtual screening for the discovery of bioactive natural products [@doi:10.1007/978-3-7643-8117-2_6]
Integrated in Silico Tools for Exploiting the Natural Products’ Bioactivity [@doi:10.1055/s-2006-941506]

`//TODO`{.red}.

### The "-omics"

`//TODO`{.red}.

The object of study of metabolomics is the *metabolome*, the same way as genomics studies the *genome* , transcriptomics the *transcriptome* and proteomics the *proteome*.
While the *genome*, *transcriptome*, and *proteome* are relatively restricted in terms of diversity, it is not the case of the *metabolome*.
This is due to the number of available building blocks of each *-ome*.

A *genome* is the complete set of genetic information in an organism [@url:https://www.nature.com/scitable/definition/genome-43]. 
This information is encoded in [DNA](#dna) as a sequence of four nucleotides.
These nucleotides are adenine (A), guanine (G), cytosine (C), and thymine (T) [@doi:10.1201/9781315735368].
Small sections of [DNA](#dna), called genes, code for the [RNA](#rna) and proteins.

The full range of [RNA](#rna) expressed by a *genome* is known as its *transcriptome* [@url:https://www.nature.com/scitable/definition/transcriptome-296].
In the case of [RNA](#rna), uracil (U) replaces thymine (T) in the sequence of nucleotides [@doi:10.1201/9781315735368].

A *proteome* is the complete set of proteins expressed by an organism [@url:https://www.nature.com/scitable/definition/proteome-297].
These proteins are made of 21 building blocks, called amino acids.

`//TODO`{.red}.

Develop harder to define metabolome because of the complexity to define building blocks.

The turnover rate of metabolites is over two magnitudes higher than the ones of genes and proteins, and lies in the range of seconds [@doi:10.1021/ac8016899].

#### Metabolomics

`//TODO`{.red}.

ms-metabolomics [@doi:10.1074/jbc.R111.238691]

number 2 [@doi:10.1002/cbic.200500151]

High-throughput discovery metabolomics [@doi:10.1016/j.copbio.2014.08.006]

Defining the Metabolome: Size, Flux, and Regulation [@doi:10.1016/j.molcel.2015.04.021]

anNET: a tool for network-embedded thermodynamic analysis of
quantitative metabolome data [@doi:10.1186/1471-2105-9-199]

The million metabolome [@doi:10.1021/acs.chemrestox.6b00179]

Computational approaches for systems metabolomics [@doi:10.1016/j.copbio.2016.04.009]

Wishart progress computational metabolomics [@doi:10.1093/bib/bbm030]

Current approaches and challenges for the metabolite profiling of complex natural extracts [@doi:10.1016/j.chroma.2014.10.091]

Accelerating Metabolite Identification in Natural Product Research: Toward an Ideal Combination of Liquid Chromatography–High-Resolution Tandem Mass Spectrometry and NMR Profiling, in Silico Databases, and Chemometrics [@doi:10.1021/acs.analchem.8b05112]

The evolution of analytical chemistry methods in foodomics [@doi:10.1016/j.chroma.2015.09.007]

From chromatogram to analyte to metabolite. How to pick horses for courses from the massive web resources for mass spectral plant metabolomics [@doi:10.1093/gigascience/gix037]

Specialized Plant Metabolites: Diversity and Biosynthesis [@doi:10.1002/9783527686063.ch2]

The WEIZMASS spectral library for high-confidence metabolite identification [@doi:10.1038/ncomms12423]

From waste products to ecochemicals: Fifty years research of plant secondary metabolism [@doi:10.1016/j.phytochem.2007.09.017]

Mass spectra-based framework for automated structural elucidation of metabolome data to explore phytochemical diversity [@doi:10.3389/fpls.2011.00040]

Mass Spectrometry and Informatics: Distribution of Molecules in the PubChem Database and General Requirements for Mass Accuracy in Surface Analysis [@doi:10.1021/ac200067s]

Assessment of Metabolome Annotation Quality: A Method for Evaluating the False Discovery Rate of Elemental Composition Searches [@doi:10.1371/journal.pone.0007490]

Recent progress in the development of metabolome databases for plant systems biology [@doi:10.3389/fpls.2013.00073]

RIKEN tandem mass spectral database (ReSpect) for phytochemicals: A plant-specific MS/MS-based data resource and database [@doi:10.1016/j.phytochem.2012.07.007]

Proposed quantitative and alphanumeric metabolite identification metrics [@doi:10.1007/s11306-014-0739-6]

MS/MS spectral tag-based annotation of non-targeted profile of plant secondary metabolites [@doi:10.1111/j.1365-313X.2008.03705.x]

Prediction of HPLC Retention Index Using Artificial Neural Networks and IGroup E-State Indices [@doi:10.1021/ci9000162]

MetaboAnalyst 2.0—a comprehensive server for metabolomic data analysis [@doi:10.1093/nar/gks374]
OTHERS

A Modification of the Jaccard–Tanimoto Similarity Index for Diverse Selection of Chemical Compounds Using Binary Strings [@doi:10.1198/004017002317375064]

MetaDB a data processing workflow in untargeted MS-based metabolomics experiments [@doi:10.3389/fbioe.2014.00072]

The future of metabolic phytochemistry: larger numbers of metabolites, higher resolution, greater understanding [@doi:10.1016/j.phytochem.2007.07.010]

Plant metabolomics: large-scale phytochemistry in the functional genomics era [@doi:10/ch97dw]

Metabolite annotations based on the integration of mass spectral information [@doi:10.1111/j.1365-313X.2008.03434.x]

Current practice of liquid chromatography-mass spectrometry in metabolomics and metabonomics [@doi:10.1016/j.jpba.2013.06.032]

Selection and characterization of botanical natural products for research studies: a NaPDI center recommended approach [@doi:10.1039/C8NP00065D]

Dereplication strategies in natural product research: How many tools and methodologies behind the same concept? [@doi:10.1007/s11101-015-9448-7]

Dereplication: racing to speed up the natural products discovery process [@doi:10.1039/c4np00134f]

CAMERA: An Integrated Strategy for Compound Spectra Extraction and Annotation of Liquid Chromatography/Mass Spectrometry Data Sets [@doi:10.1021/ac202450g]

In silico prediction and automatic LC-MSn annotation of green tea metabolites in urine [@doi:10.1021/ac403875b]

HERMES: a molecular-formula-oriented method to target the metabolome [@doi:10.1038/s41592-021-01307-z]

xMSannotator: An R package for network- based annotation of high-resolution metabolomics data [@doi:10.1021/acs.analchem.6b01214]

iMet: A Network-Based Computational Tool to Assist in the Annotation of Metabolites from Tandem Mass Spectra [@doi:10.1021/acs.analchem.6b04512]

Identifying metabolites by integrating metabolome databases with mass spectrometry cheminformatics [@doi:10.1038/nmeth.4512]

Strategies and Challenges in Method Development and Validation for the Absolute Quantification of Endogenous Biomarker Metabolites Using Liquid Chromatography-Tandem Mass Spectrometry [@doi:10.1002/mas.21607]

High-throughput mass spectrometry scheme for screening and quantification of flavonoids in antioxidant nutraceuticals [@doi:10.1016/j.chroma.2019.460408 ]

Multi-omics integration in biomedical research – A metabolomics-centric review [@doi:10.1016/j.aca.2020.10.038]

<!-- ADLIN
1.6.2.4 MS-based metabolomics
Analysis of high-throughput NP extracts using methods such as LC-HRMS requires systematic procedures and for this, the generic metabolomics workflow commonly used for clinical samples can also be applied to NPs. Metabolomics is mainly associated with the high
20
 Chapter 1
throughput ability of modern analytical techniques combined with the statistical power of multivariate data analysis (MVDA) (Wolfender et al., 2013a). Such studies aimed at quantification and identification of small metabolites (kDa) in biological samples, including those from herbal materials (Lajis et al., 2017; Wolfender et al., 2013a). Apart from LC-MS (Wolfender et al., 2015), other analytical platforms such as Fourier transform-infrared spectroscopy (FT-IR) (Abdullah et al., 2012; Lee et al., 2017), and nuclear magnetic resonance (NMR) (Kim et al., 2010; Mcalpine et al., 2019; Robinette et al., 2012) are some of the recognized tools in plant metabolomics.
There are two types of LC-HRMS-based metabolomics experiments, including untargeted and targeted experiments. Untargeted or discovery-based metabolomics provides comprehensive detection and relative quantitation of small molecules in complex mixtures (Schrimpe-Rutledge et al., 2016), which generally followed the workflow illustrated in Figure 1.2.
Figure 1.2 Schematic of untargeted MS-based metabolomics.
For untargeted MS metabolomics, information on MS/MS fragmentation is needed for metabolite annotation or identification. MS/MS features can be acquired automatically either by data-dependent (DDA) or data-independent (DIA) acquisition (Wolfender et al., 2015). DDA involved performing a full scan survey in a given sample, followed by a series of MS/MS scans that are triggered when ions of specific signals (e.g. intensity, charge state) are detected in the full scan (Eliuk & Makarov, 2015; Kind et al., 2018). By contrast, DIA does not use a selection step prior to fragmentation and offer greater MS/MS coverage than DDA (Eliuk & Makarov, 2015; Wolfender et al., 2015).
Targeted or validation-based metabolomics is aimed at measuring already known groups of metabolites with opportunities for absolute quantitation (Schrimpe-Rutledge et al., 2016). Targeted metabolomics usually utilized triple-quadrupole (QqQ) MS/MS system with multiple reaction monitoring (MRM) experiments (Wolfender et al., 2013a). In MRM, precursor ions corresponding to the metabolites of interest are selected in the first quadrupole from among various ions produced by the ionisation probe, the precursor ions are then fragmented in the
 21

 Chapter 1
collision cell (second quadrupole) into product ions, where one or more product ions can be selected for quantitation (third quadrupole) (Wei et al., 2010).
After MS pre-processing step, the resulting large data tables demand the use of multivariate statistics to reveal hidden information (Kellogg et al., 2016; Okada et al., 2010). For this, unsupervised methods such as principal component analysis (PCA) and hierarchal clustering analysis (HCA) are usually used as a starting point for MVDA, particularly for explorative analysis and outlier detection (Boccard et al., 2010). PCA is a dimensionality-reduction method that transforms a large set of variables to a lower-dimensional space of uncorrelated variables called principal components (PCs) (Okada et al., 2010). Transformation of the original data (detected peaks in chemical profile) into the new coordinate system or PC preserves as much variance (in the first principal component) in the original data as possible and thus highlighting the largest differences between samples (Kellogg et al., 2016; Rajalahti & Kvalheim, 2011). Cluster analysis such as HCA is used mainly in conjunction with PCA to aid sample classifications. HCA can assist to partition a data set into natural groups if no class or response is expected (Boccard et al., 2010). This type of analysis may use agglomerative methods to group similar observations into groups called clusters by computing the distance between sets of observations, which are visualized with a dendrogram (Boccard et al., 2010; Gries, 2015).
The next step after the untargeted analysis is typically comprised of supervised methods. Partial least square discriminant analysis (PLS-DA) is one of the most common supervised methods used for feature selection and classifications (Szyman ́ska et al., 2012). This method optimised the separations between sample groups by linking two data matrices, X (e.g. spectral data) and Y, the groups/class membership used for prediction. PLS-DA is an extension of the multiple linear regression of partial least square (PLS) model, which can handle multiple dependent categorical variables. PLS-DA aims at maximizing the covariance between the X and the corresponding Y-variables of highly multidimensional data by finding a linear subspace of the explanatory variables (Gromski et al., 2015).
1.6.2.5 Metabolomics application in herbal authentication
Metabolomics has been evolving extensively over the past years, with great prospects in many fields of life sciences and natural product research. Such developments were also beneficial for the quality control of medicinal plants (Lajis et al., 2017; Pelkonen et al., 2012). For example, Kim et al (2011) positively utilized UPLC-QTof MS and MVDA (HCA and PLS-DA) to classify P. ginseng samples aged 1 to 3 and applied metabolite selections to precisely classify the highest
22

Chapter 1
demand samples aged 4, 5 and 6 (N. Kim et al., 2011). Kellog et al (2017) evaluated untargeted UHPLC-MS, targeted UHPLC-MS and untargeted quantitative 1H-NMR metabolomics using PCA model to investigate how different metabolomics approaches facilitate the discrimination of 34 commercialized Camellia sinensis (green tea), categorized as tea or supplement products (Kellogg et al., 2017). Amongst these three methods, untargeted UHPLC-MS (2270 marker ions) was shown to be the most effective at discriminating between tea leaf, tea powder, supplement, and non-green-tea products. While NMR metabolomics displayed similar trends as the untargeted MS, there were more overlaps between tea leaf and tea powder. On the contrary, the targeted UHPLC-MS methods, which quantified only 16 green tea markers, failed to discriminate between teas and supplement samples. MS metabolomics combined with PCA and HCA has also been successful to differentiate 55 bitter melon seeds collected in different parts of China into two chemotypes (S. Zhou et al., 2019). The authors further selected 28 discriminatory markers by limiting the threshold scores in the contribution plot. The two chemotypes were identified as either rich in cucurbitane or oleanane triterpenoid glycosides on the basis of marker identification using six standards or annotation of the remaining markers via Molecular networking (MN)-based and in-silico dereplication (S. Zhou et al., 2019). -->


<!-- 
ABDUL
1.5 Metabolomics approach in natural products research
In natural products research, metabolites are compounds, which can be an intermediate or end product of a metabolism process. There are divided into two groups, the first one is the “primary metabolites”, which consist of compounds that are considered essential for life and reproduction. The second group is “secondary metabolites” which serve complimentary functions such as defense, reproduction and signaling.128 However, since many of primary metabolism intermediates overlap with secondary metabolite intermediates, the distinction between primary and secondary metabolites is sometimes unclear. For instance, sterols are considered mainly as secondary metabolites but in fact they are part of the cell structural framework.193 Metabolomics is a holistic approach that is used mainly to monitor the metabolome changes in a group of biological samples and to compare patterns of metabolites in a given biological system.194 In some studies, metabolomics refer also to the comprehensive analysis of all metabolites in a given organism (deep metabolome analysis). This approach was firstly introduced in 1990s as a result of technological breakthrough in the investigation of natural products of small molecular weight. It aims to provide a comprehensive qualitative and quantitative survey of organic small molecules (<1000 Da) within various biological matrices.
Chemical profiling aims to attain either a brief or a detailed description of the chemical content of given extracts or biological fluids. The concept of chemical profiling has been reported as early as 1971 to monitor sugar and drug metabolites in human fluids using GC-MS.195 Since then, research in this field has considerably expanded and includes various type of biological matrices and notably in the field of NP research, plant and microbial extracts. The corresponding analytical techniques have also been significantly developed and allow efficient profiling of a wide range of major and minor small molecules. They also provide valuable spectroscopic information for their annotation. However, the precise identification of all metabolites produced by a living organism is still challenging to date, especially when using a single analytical technique or lacking reference spectroscopic database and chemical standards.196
The three main methods for chemical profiling can be classified as follows :
The first one is a rapid and superficial method which known as “metabolomic fingerprinting”.197 It provides chemical classification of different samples by comparing metabolic patterns and highlighting significant differences usually at the level of spectroscopy features. This kind of analysis is usually performed either by spectral methods without hyphenation (e.g., IR, NMR) or by fast LCMS with low chromatographic resolution and high throughput. The interpretation of the data in this method is performed by subsequent multi-variate data analyses. An example of this
41
method is the exploration of pathogen-specific metabolic fingerprints, which has allowed for the rapid detection of pathogenic organisms.198 In this study, 100 clinical isolates representing 10 types of infections were analyzed by GCMS and more than 800 MS features were compared. Multi- variate data analyses revealed a distinguished fingerprint for each pathogen group with 95% percent accuracy for group differentiation.198
The second method is known as "metabolic profiling" which allows deep analysis and extensive profiling of one or more samples in a given biological state. In contrast to metabolic fingerprinting, it aims to identify individual metabolites present in the sample (e.g., biomarkers in relation to disease, stress-state signaling, genetic modifications ... etc.). This approach is generally performed by LCMS at high resolution using optimized high resolution chromatographic conditions and highly sensitive spectrometric methods. The ability to confidently identify metabolites is essential. In modern LCMS platforms, metabolite profiling is typically performed with data-dependent MS/MS analysis that generates for most LC peaks MS1 data for molecular weight determination and subsequent molecular formula assignment and corresponding MS/MS fragmentation spectra for subsequent annotation.199 In such an approach, adapted processing techniques and data mining tools are also essential. One example is the use of untargeted tandem LC-MS combined with molecular networking (MN) and principal component analysis (PCA) to identify metabolites released by the fungal pathogen Ilyonectria mors-panacis that causes ginseng root rot.200 In this study, the ethyl acetate extracts of 22 fungal strains were analyzed by LC-MS/MS. This allowed the confident identification of decay-causing compounds without the need for standards.
The third method is targeted analysis. It enables the precise quantification of biomarkers of interest and is referred to as "metabolites target analysis". Moreover, it is used to precisely monitor one or several biomarkers, which are identified unambiguously by several approaches to establish a correlation between the biomarker levels and some metabolic pathways or environmental stresses. A study was done by Xu and co-authors investigated the metabolic response to visible light exposure in three Lactobacillus species through the application of LCMS-based targeted metabolomics.201 As a result, 12 metabolites showed significant differences in light exposure culture compared to the control samples.201
In most cases, metabolomics is used as an untargeted analysis workflow to provide high metabolite coverage. However, this approach requires significant effort to process and interpret the data. Once clues to metabolite type are known, semi-targeted metabolomics methods can be effectively used to explore metabolite induction or other variation in biomarker concentrations. As mentioned, once the biomarker is known, targeted approaches can be used to provide absolute quantification by comparing them to authentic standards.202 This approach is characterized by low complexity data processing, but it is limited by the fact that it requires the construction of a dedicated quantification method. In some workflows, untargeted analysis can be replaced by semi-targeted metabolomic approach where a rather large set of predefined metabolites has to be monitored. In particular, these approaches are used for the analysis of body fluids. For example, a study was designed to pursue
42

analytical coverage of a predefined list of plasma metabolites consisting of 270 molecules to characterize the micronutrient status of human blood. LC–MS and GC–MS analyses allowed the detection of several classes of metabolites that were not easily detected by untargeted analyses.203
Metabolomics approaches have emerged in the early 2000s in many fields of life science and become used more and more in many fields such as agriculture (e.g., plant response to stress), botany (e.g., chemotaxonomy of related plant species), medicine (e.g., diseases biomarker discovery), and pharmaceutical science (e.g., drug discovery, quality control). In NP research, related to the field of the pharmaceutical science, metabolites profiling and metabolomics are used notably for in depth dereplication studies, authentication of phyto-pharmaceutical preparations and detection of possible adulterations.204
1.5.1 Sample preparation
Metabolomics in NP studies involves a wide variety of natural extracts, which are obtained from different biological matrices. Examples of these biological matrices include tissues of plant or animal origins, cultures of bacteria or fungi and fluids of animal secretions or plant exudates. This diversity requires the development of adapted sample preparation methods that aim at extracting the largest possible number of metabolite while ensuring a good metabolite profiling reproducibility and compatibility with the used analytical platform.205 Overall, the different methods of sample preparation are mainly aimed at creating representative samples.
Depending on the biological problem to be solved, the sampling procedure, number of replicates, and timing of collection have to be well defined to ensure a comprehensive gathering of information. The required amount of the sample can be discussed according to the goal of the study and the sensitivity of the instrument. For example, few hundreds of micrograms are sufficient for LCMS analysis, but when using of LC-NMR, samples are required to be in milligram amounts to meet the inherent low level of NMR sensitivity. Overall, the sample preparation process typically involves three steps: harvesting, extraction then preparation for analysis.
The collection of a sample at its origin must follow certain recommendations to avoid possible contamination and decomposition of the contents. First of all, in most of the cases, the transfer of the collected material should be done under cold conditions by using dry ice or liquid nitrogen to prevent enzymatic degradation and to avoid oxidation. Storage of samples are also preferred to be in cold and dry environment, the usage of -80 °C freezers are recommended for long term storage.206 In the case of plants, the harvest time of the year and sometimes the day must be taken into account, as the level of metabolites varies according to the period.207 The organ or the part of the organism should be taken into account, different parts may not mixed together, e.g., young and old leaves should be separated. In the case of microorganisms, the cultivation time must be considered in relation to the growth cycle and the harvesting is often recommended at the time of maximum growth.
43

During the extraction step, several organic solvents can be used to capture the desired content. The choice of solvent depends directly on the type of targeted compound. As a general rule, "like attracts like", fatty acids and terpenes can be extracted with hydrophobic solvents like hexane. Middle polar compounds are dissolved in dichloromethane and ethyl acetate. Sugars and polar compounds can be extracted with hydroalcoholic solutions. In the case of microorganism culture, ethyl acetate is the most commonly used solvent for the extraction of secondary metabolites.208 For the extraction of solid material, the smaller the particle size, the better yield due to increase in surface area exposed to the solvent.209 The solvent to solute ratio must be considered to avoid solvent saturation, which limits the ability to extract the maximum amount of metabolites. In some cases, after a primary extraction performed by direct agitation, a liquid-liquid partition is necessary to recover an extract enriched in secondary metabolites and remove most of the undesirable compounds such as sugars and fatty acids. For this use of chloroform and methanol-water in a certain proportion is widely accepted method. Due to certain safety issues associated with chloroform, methyl tert-butyl ether (MTBE) or n-hexane can be used as a valid alternative.
Prior to analysis, samples must be fully dissolved in a suitable solvent and concentrated to certain levels depending on the detector used. Furthermore, the elimination of unwanted substances and impurities can help to reduce the experimental errors. Solid phase extraction (SPE) is a common approach in phytochemical research.210 Because SPE is an environmentally friendly and relatively inexpensive technique, it is preferred over liquid-liquid extraction (LLE) and has become a popular method in recent years. Several types of stationary phases can be chosen according to experiment goal. For example, apolar silica gel (C2, C8, C18) are suitable for reverse phase, while unbounded silica, cyanopropyl and neutral alumina are recommended for normal phase.210 Upon introduction of the sample in the SPE cartridge, a small amount of eluent is added, and a vacuum is applied to elute the mixture and the mobile phase through the cartridge. Depending on the polarity of the solid phase and the chosen solvent, the targeted compounds are trapped in the solid phase and the unwanted substances are released first or vice versa. Commercially available SPE cartridges have limited capacity to adsorb matrices up to 100 mg. If higher amount is needed, the same setup can be upscaled to vacuum liquid chromatography (VLC) increasing the capacity to reach 1-2 g of loading sample. Overall, the major issue in working with biological raw materials is the fact that they are extracted from living organisms, which are affected by the surrounded environmental factors. Changes in their environment, which can also be characterized by metabolomics, are led to significant qualitative and quantitative variations in the production of secondary metabolites. For studies that compare the metabolome of related organisms, careful attention to the control of all bio-sources and experimental conditions is required to validate the reproducibility of the experiments.
 -->
### Open Science

`//TODO`{.red}.

Formatting Open Science: agilely creating multiple document formats for academic manuscripts with Pandoc Scholar [@doi:10.7717/peerj-cs.112]

How to make models more useful [@doi:10/h9kz]

Wikidata and the bibliography of life [@doi:10.7717/peerj.13712]

A Quick Guide to Software Licensing for the Scientist-Programmer [@doi:10.1371/journal.pcbi.1002598]

#### Data Sharing

`//TODO`{.red}.

#### Metadata

`//TODO`{.red}.

### Technologies

`//TODO`{.red}.

#### Databases

`//TODO`{.red}.

Tsugawa databases deepen understanding of metabolisms [@doi:10.1016/j.copbio.2018.01.008]

FlavonQ [@doi:10.1021/acs.analchem.5b02624]
FlavonQ DB [@doi:10/h83p]

#### Ontologies

`//TODO`{.red}.

#### Knowledge Graphs

`//TODO`{.red}.

#### Machine Learning

`//TODO`{.red}.