## Computational Era {#computational}

Switch to the macroscope

Cybernetics/Systemics

`//TODO`{.red}.

### The "-omics"

`//TODO`{.red}.

The object of study of metabolomics is the *metabolome*, the same way as genomics studies the *genome* , transcriptomics the *transcriptome* and proteomics the *proteome*.
While the *genome*, *transcriptome*, and *proteome* are relatively restricted in terms of diversity, it is not the case of the *metabolome*.
This is due to the number of available building blocks of each *-ome*.

A *genome* is the complete set of genetic information in an organism [@url:https://www.nature.com/scitable/definition/genome-43]. 
This information is encoded in [DNA](#dna) as a sequence of four nucleotides.
These nucleotides are adenine (A), guanine (G), cytosine (C), and thymine (T) [@doi:10.1201/9781315735368].
Small sections of [DNA](#dna), called genes, code for the [RNA](#rna) and proteins.

The full range of [RNA](#rna) expressed by a *genome* is known as its *transcriptome* [@url:https://www.nature.com/scitable/definition/transcriptome-296].
In the case of [RNA](#rna), uracil (U) replaces thymine (T) in the sequence of nucleotides [@doi:10.1201/9781315735368].

A *proteome* is the complete set of proteins expressed by an organism [@url:https://www.nature.com/scitable/definition/proteome-297].
These proteins are made of 21 building blocks, called amino acids.

Develop harder to define metabolome because of the complexity to define building blocks.

The turnover rate of metabolites is over two magnitudes higher than the ones of genes and proteins, and lies in the range of seconds [@doi:10.1021/ac8016899].

#### Metabolomics

`//TODO`{.red}.

The million metabolome [@doi:10.1021/acs.chemrestox.6b00179]

Computational approaches for systems metabolomics [@doi:10.1016/j.copbio.2016.04.009]

Wishart progress computational metabolomics [@doi:10.1093/bib/bbm030]

Current approaches and challenges for the metabolite profiling of complex natural extracts [@doi:10.1016/j.chroma.2014.10.091]

Accelerating Metabolite Identification in Natural Product Research: Toward an Ideal Combination of Liquid Chromatography–High-Resolution Tandem Mass Spectrometry and NMR Profiling, in Silico Databases, and Chemometrics [@doi:10.1021/acs.analchem.8b05112]

The evolution of analytical chemistry methods in foodomics [@doi:10.1016/j.chroma.2015.09.007]

From chromatogram to analyte to metabolite. How to pick horses for courses from the massive web resources for mass spectral plant metabolomics [@doi:10.1093/gigascience/gix037]

Specialized Plant Metabolites: Diversity and Biosynthesis [@doi:10.1002/9783527686063.ch2]

The WEIZMASS spectral library for high-confidence metabolite identification [@doi:10.1038/ncomms12423]

From waste products to ecochemicals: Fifty years research of plant secondary metabolism [@doi:10.1016/j.phytochem.2007.09.017]

Mass spectra-based framework for automated structural elucidation of metabolome data to explore phytochemical diversity [@doi:10.3389/fpls.2011.00040]

Mass Spectrometry and Informatics: Distribution of Molecules in the PubChem Database and General Requirements for Mass Accuracy in Surface Analysis [@doi:10.1021/ac200067s]

Assessment of Metabolome Annotation Quality: A Method for Evaluating the False Discovery Rate of Elemental Composition Searches [@doi:10.1371/journal.pone.0007490]

Recent progress in the development of metabolome databases for plant systems biology [@doi:10.3389/fpls.2013.00073]

RIKEN tandem mass spectral database (ReSpect) for phytochemicals: A plant-specific MS/MS-based data resource and database [@doi:10.1016/j.phytochem.2012.07.007]

Proposed quantitative and alphanumeric metabolite identification metrics [@doi:10.1007/s11306-014-0739-6]

MS/MS spectral tag-based annotation of non-targeted profile of plant secondary metabolites [@doi:10.1111/j.1365-313X.2008.03705.x]

Prediction of HPLC Retention Index Using Artificial Neural Networks and IGroup E-State Indices [@doi:10.1021/ci9000162]

MetaboAnalyst 2.0—a comprehensive server for metabolomic data analysis [@doi:10.1093/nar/gks374]
OTHERS

A Modification of the Jaccard–Tanimoto Similarity Index for Diverse Selection of Chemical Compounds Using Binary Strings [@doi:10.1198/004017002317375064]

MetaDB a data processing workflow in untargeted MS-based metabolomics experiments [@doi:10.3389/fbioe.2014.00072]

The future of metabolic phytochemistry: larger numbers of metabolites, higher resolution, greater understanding [@doi:10.1016/j.phytochem.2007.07.010]

Plant metabolomics: large-scale phytochemistry in the functional genomics era [@doi:10/ch97dw]

Metabolite annotations based on the integration of mass spectral information [@doi:10.1111/j.1365-313X.2008.03434.x]

Current practice of liquid chromatography-mass spectrometry in metabolomics and metabonomics [@doi:10.1016/j.jpba.2013.06.032]

Selection and characterization of botanical natural products for research studies: a NaPDI center recommended approach [@doi:10.1039/C8NP00065D]

Dereplication strategies in natural product research: How many tools and methodologies behind the same concept? [@doi:10.1007/s11101-015-9448-7]

Dereplication: racing to speed up the natural products discovery process [@doi:10.1039/c4np00134f]

CAMERA: An Integrated Strategy for Compound Spectra Extraction and Annotation of Liquid Chromatography/Mass Spectrometry Data Sets [@doi:10.1021/ac202450g]

In silico prediction and automatic LC-MSn annotation of green tea metabolites in urine [@doi:10.1021/ac403875b]

HERMES: a molecular-formula-oriented method to target the metabolome [@doi:10.1038/s41592-021-01307-z]

xMSannotator: An R package for network- based annotation of high-resolution metabolomics data [@doi:10.1021/acs.analchem.6b01214]

iMet: A Network-Based Computational Tool to Assist in the Annotation of Metabolites from Tandem Mass Spectra [@doi:10.1021/acs.analchem.6b04512]

Identifying metabolites by integrating metabolome databases with mass spectrometry cheminformatics [@doi:10.1038/nmeth.4512]

Strategies and Challenges in Method Development and Validation for the Absolute Quantification of Endogenous Biomarker Metabolites Using Liquid Chromatography-Tandem Mass Spectrometry [@doi:10.1002/mas.21607]

High-throughput mass spectrometry scheme for screening and quantification of flavonoids in antioxidant nutraceuticals [@doi:10.1016/j.chroma.2019.460408 ]


TEST

[@agrees_with:10.1371/journal.pcbi.1007128]

### Open Science

`//TODO`{.red}.

Formatting Open Science: agilely creating multiple document formats for academic manuscripts with Pandoc Scholar [@doi:10.7717/peerj-cs.112]

#### Data Sharing

`//TODO`{.red}.

#### Metadata

`//TODO`{.red}.

### Technologies

`//TODO`{.red}.

#### Databases

`//TODO`{.red}.

Tsugawa databases deepen understanding of metabolisms [@doi:10.1016/j.copbio.2018.01.008]

FlavonQ [@doi:10.1021/acs.analchem.5b02624]
FlavonQ DB [@doi:10/h83p]

#### Ontologies

`//TODO`{.red}.

#### Knowledge Graphs

`//TODO`{.red}.

#### Machine Learning

`//TODO`{.red}.