## Data & Computational Era {#sec:computational}

The classical reductionist approach detailed above, allowing, from a complex organism, to scrutinize the smallest quantitative change of a single metabolite is impressive, both by its precision and by the technologies used.
Along the way, large amounts of data are generated, capturing much more than these important, but very targeted changes.
And it is precisely this data that can extend the meticulous analysis to a broader context.

This extension has already been illustrated by Joël de Rosnay in his book, *Le Macroscope - Vers une vision globale* [@isbn:2-02-002818-2].
This book advocates a *systemic* approach to complement the classical analytical approach.
Influenced by *cybernetics*, or how a system senses and reacts to a stimulus, the book describes three main instruments that should be used in Science:
The first instrument is the *microscope*, allowing one to study the *infinitely small*, which in the field of phytochemistry could correspond to the various detectors listed before.
The second one is the *telescope*, allowing one to study the *infinitely large*.
This instrument is mainly used in physics and not really in biology or chemistry, unlike the last instrument, the *macroscope*.
The *macroscope* is a symbolical instrument allowing one to study the *infinitely complex*.
This symbolic instrument is very similar to modern computers.

To that extent, computers now allow the study of the *relatively complex*, and adequate tools and resources must be developed to achieve a broader picture of what is captured by targeted experiments [@doi:10.1126/science.1142502].

### The «-omics»

This broader picture is also the goal of the different "-omics" fields.
Their holistic approach focuses on complex interactions within or between biological systems [@doi:10.1042/EBC20180003].
They are now used across biology-related fields, before or in complement to targeted analysis.
The central dogma of molecular biology is that [DNA](#dna) makes [RNA](#rna) and [RNA](#rna) makes proteins [@doi:10.1371/journal.pbio.2003243].
The complexity of processes involved is slightly higher than that, and therefore, many proposals have been made to remodel it.
Among them, a proposal was made to place *metabolomics* at the center of other "-omics", in a multiinteraction dynamic perspective [@doi:10.1007/s11306-021-01800-8].
The object of study of metabolomics is the *metabolome*, the same way as genomics studies the *genome*, transcriptomics the *transcriptome*, and proteomics the *proteome*.
While the *genome*, *transcriptome*, and *proteome* are relatively restricted in terms of diversity, it is not the case with the *metabolome*.
This is due to the number of available building blocks of each *-ome*.

A *genome* is the complete set of genetic information in an organism [@url:https://www.nature.com/scitable/definition/genome-43]. 
This information is encoded in [DNA](#dna) as a sequence of four nucleotides.
These nucleotides are adenine (A), guanine (G), cytosine (C), and thymine (T) [@doi:10.1201/9781315735368].
Small sections of [DNA](#dna), called genes, code for [RNA](#rna) and proteins.

The full range of [RNA](#rna) expressed by a *genome* is known as its *transcriptome* [@url:https://www.nature.com/scitable/definition/transcriptome-296].
In the case of [RNA](#rna), uracil (U) replaces thymine (T) in the sequence of nucleotides [@doi:10.1201/9781315735368].

A *proteome* is the complete set of proteins expressed by an organism [@url:https://www.nature.com/scitable/definition/proteome-297].
These proteins are made of 22 building blocks [@doi:10.1038/nchembio847], called amino acids, of which 21 are found in eurkaryotes.

Therefore, studying the *metabolome*, or "all small molecule (<1500 Da) metabolites found in a specific cell, organ or organism" [@doi:10.1093/nar/gkl923], appears more challenging, as they are not restricted by such building blocks.
Moreover, the turnover rate of metabolites is over two magnitudes higher than the ones of genes and proteins and lies in the range of seconds [@doi:10.1021/ac8016899].
Taking all these considerations into account, metabolomics, its holistic approach and its close link with bioinformatics will be described below.

#### Metabolomics

In the same way that it studies its objects of analysis, metabolomics encompasses a whole series of different tools, from data acquisition to its treatment, statistical processing and biological interpretation [@doi:10/ch97dw; @doi:10.1016/j.jare.2014.10.003].
While metabolomics appproaches are not limited to [MS](#ms) [@doi:10/dgtr9s], the most of them are performed with [MS](#ms).
This is due to the good compromise [MS](#ms) offers in terms of sensitivity, speed [@doi:10.1016/j.copbio.2014.08.006] and selectivity [@doi:10.1074/jbc.R111.238691; @doi:10.1002/cbic.200500151]. 
As [MS](#ms) detectors and modes of acquistion were detailed before, current section will focus on the data generated by [MS](#ms)-based metabolomics and the tools and resources required to extract biological knowledge out of them. 

##### Data Conversion

The first step after data acquisition is generally to convert the obtained file to an open format.
This step is crucial, because it allows then to exchange data between laboratories, and to process them with any software, open or not.
The currently most used software in this resect is MSConvert [@doi:10.1038/nbt.2377].
Years after its develoment, other solutions to convert the data have been developed, but with a more focused audience, either because of the language they use [@doi:10.1021/acs.jproteome.0c00866], or their abiity to convert the data of limited instruments only [@doi:10.1021/acs.jproteome.9b00328].

##### Data Processing

After conversion, the next major step is to process the obtained data.
Starting from a *chromatogram*, this step will produce a *feature list*, with associated *spectra* and metadata [@doi:10.1093/gigascience/gix037].
The main relevant terms of this step are illustrated in Figure @fig:intro-feature.

![**Common terminology used in metabolomics**. The notions of *chromatogram*, *spectrum*, *peak*, *feature*, and *feature list (or table)* are illustrated.](images/intro-feature.pdf "intro-feature"){#fig:intro-feature short-caption="Common Terminology Used in metabolomics" align="center" width="100%"}

The initial object in Figure @fig:intro-feature is a *chromatogram*.
It corresponds to a serie of intensities as a function of the retention time ([RT](#rt)).
In the case of [MS](#ms), this chromatogram hides a third dimension, the mass-to-charge ratio, or [*m/z*](#m/z) ([PDA](#pda) also holds a third dimension, the wavelength).
This third dimension is used to define a *spectrum*.

A spectrum is a serie of intensities as a function of the [*m/z*](#mz), at a given timepoint.
This spectrum can eventually be combined with other spectra to lead to a *pseudospectrum*.
In the case of [MS](#ms) and [MS/MS](#msms) acquisition, different spectra co-exist on a same chromatogram (*full scan* and *tandem* spectra).

The first processing step is therefore, from the initial data, to re-construct several chromatograms, one for each [*m/z*](#mz) (with a given tolerance).
From these re-constructed chromatograms, different algorithmic tools exist to extract *peaks* from a chromatogram [@doi:10.1021/ac051437y; @doi:10.1021/ac202450g; @doi:10.1021/acs.analchem.9b01424].
Depending on their complexity, the algorithms can consider two or three dimensions to extract the peaks.
This step is generally called *deconvolution*.

A peak consists of a serie of descriptive parameters related to an [*m/z*](#m/z) and an [RT](#rt).
Classically, most common parameters describe the peak intensity, its area, its minimal and maximal [RT](#rt)s and [*m/z*](#m/z)s, the [RT](#rt) of its apex (global maximum), and its Full Width at Half Maximum ([FWHM](#fwhm)).
Other parameters related to the peak shape can be used such as the asymetry and tailing factors.

<!-- 

JL bentham science [@doi:10.2174/092986713805288932]

Defining the Metabolome: Size, Flux, and Regulation [@doi:10.1016/j.molcel.2015.04.021]

The million metabolome [@doi:10.1021/acs.chemrestox.6b00179]

Computational approaches for systems metabolomics [@doi:10.1016/j.copbio.2016.04.009]

Wishart progress computational metabolomics [@doi:10.1093/bib/bbm030]

Current approaches and challenges for the metabolite profiling of complex natural extracts [@doi:10.1016/j.chroma.2014.10.091]

Specialized Plant Metabolites: Diversity and Biosynthesis [@doi:10.1002/9783527686063.ch2]

MetaboAnalyst 2.0—a comprehensive server for metabolomic data analysis [@doi:10.1093/nar/gks374]
OTHERS

A Modification of the Jaccard–Tanimoto Similarity Index for Diverse Selection of Chemical Compounds Using Binary Strings [@doi:10.1198/004017002317375064]

MetaDB a data processing workflow in untargeted MS-based metabolomics experiments [@doi:10.3389/fbioe.2014.00072]

The future of metabolic phytochemistry: larger numbers of metabolites, higher resolution, greater understanding [@doi:10.1016/j.phytochem.2007.07.010]

Plant metabolomics: large-scale phytochemistry in the functional genomics era [@doi:10/ch97dw]

Current practice of liquid chromatography-mass spectrometry in metabolomics and metabonomics [@doi:10.1016/j.jpba.2013.06.032]

Selection and characterization of botanical natural products for research studies: a NaPDI center recommended approach [@doi:10.1039/C8NP00065D]

CAMERA: An Integrated Strategy for Compound Spectra Extraction and Annotation of Liquid Chromatography/Mass Spectrometry Data Sets [@doi:10.1021/ac202450g]

In silico prediction and automatic LC-MSn annotation of green tea metabolites in urine [@doi:10.1021/ac403875b]

HERMES: a molecular-formula-oriented method to target the metabolome [@doi:10.1038/s41592-021-01307-z]

xMSannotator: An R package for network- based annotation of high-resolution metabolomics data [@doi:10.1021/acs.analchem.6b01214]

iMet: A Network-Based Computational Tool to Assist in the Annotation of Metabolites from Tandem Mass Spectra [@doi:10.1021/acs.analchem.6b04512]

Strategies and Challenges in Method Development and Validation for the Absolute Quantification of Endogenous Biomarker Metabolites Using Liquid Chromatography-Tandem Mass Spectrometry [@doi:10.1002/mas.21607]

QA/QC [@doi:10.1007/s11306-022-01926-3]
 -->

`//TODO`{.red}.

Don't forget GNPS metgem

### Open Science

`//TODO`{.red}.

<!-- 
Formatting Open Science: agilely creating multiple document formats for academic manuscripts with Pandoc Scholar [@doi:10.7717/peerj-cs.112]

How to make models more useful [@doi:10/h9kz]

Wikidata and the bibliography of life [@doi:10.7717/peerj.13712]

A Quick Guide to Software Licensing for the Scientist-Programmer [@doi:10.1371/journal.pcbi.1002598] 
-->

#### Data Sharing

`//TODO`{.red}.

#### Metadata

`//TODO`{.red}.

### Technologies

`//TODO`{.red}.

#### Databases

`//TODO`{.red}.

<!-- 

Mass Spectrometry and Informatics: Distribution of Molecules in the PubChem Database and General Requirements for Mass Accuracy in Surface Analysis [@doi:10.1021/ac200067s]

Tsugawa databases deepen understanding of metabolisms [@doi:10.1016/j.copbio.2018.01.008]

RIKEN tandem mass spectral database (ReSpect) for phytochemicals: A plant-specific MS/MS-based data resource and database [@doi:10.1016/j.phytochem.2012.07.007]

FlavonQ [@doi:10.1021/acs.analchem.5b02624]
FlavonQ DB [@doi:10/h83p] 
-->

#### Ontologies

`//TODO`{.red}.

#### Knowledge Graphs

`//TODO`{.red}.

#### Machine Learning

`//TODO`{.red}.

<!-- 
Dot product [@doi:10/bhw492]

Weightings [@doi:10.1093/bioinformatics/bts083]

New methods (Spec2Vec and consorts)

Prediction of HPLC Retention Index Using Artificial Neural Networks and IGroup E-State Indices [@doi:10.1021/ci9000162]

-->

### The *Plant Metabolome*

To refine the first approximation of *plant metabolome* it is first necessary to better define the source it putatively originates from, the plant.

`//TODO`{.red}.

The small molecules can originate from other sources, such as endophytes present inside the plant.

`//TODO`{.red}.

<!-- 

Recent progress in the development of metabolome databases for plant systems biology [@doi:10.3389/fpls.2013.00073]

From waste products to ecochemicals: Fifty years research of plant secondary metabolism [@doi:10.1016/j.phytochem.2007.09.017]

microbiome effect on metabolome [@doi:10.1073/pnas.0812874106]

reading the metabolic fine print [@doi:10.1038/embor.2008.236]

in reality, holobiont [@doi:10.1016/j.mib.2017.07.001]

if needed hologenome [@doi:10.1186/s40168-018-0457-9]

big endophyte book [@doi:10.1007/978-3-319-90484-9_9]

endophytes [@doi:10.3389/fmicb.2016.01538]

number 2 [@doi:10.3389/fbioe.2020.00467] 
-->

<!-- 
NP Rollinger
Virtual screening for the discovery of bioactive natural products [@doi:10.1007/978-3-7643-8117-2_6]
Integrated in Silico Tools for Exploiting the Natural Products’ Bioactivity [@doi:10.1055/s-2006-941506] 
-->