#### Discussion

The metabolite annotation process can be boiled down to the comparison of attributes (e.g. exact mass, molecular formula ([MF](#mf)) fragmentation spectra) of the queried analyte to attributes of candidate structures present in a [DB](#db).
When [HRMS](#hrms) and appropriate heuristic filters are used, the establishment of the [MF](#mf) of the analyte is relatively straightforward [@doi:10.1186/1471-2105-8-105].
However, this is not sufficient to proceed to metabolite annotation given the isomeric nature of numerous [NP](#np): over all compounds reported in the [DNP](#dnp), less than 10% have a unique chemical formula, the average number of compounds per molecular formula is 8.6 and a maximum of 1,274 isomers is found for C~15~H~20~O~4~.
With an [MS](#ms)1 analysis relying on exact mass only, no ranking between those isomers is possible.
Computational metabolite annotation tools allow to attribute a score to candidate structures and, thus, to discriminate isomeric molecules.
However, [MF](#mf) and fragmentation spectra are not the only attributes which can be compared in the metabolite annotation process.
Specialized metabolites, as products of biosynthetic clusters themselves part of the genome, are tightly linked to the taxonomic position of the producing organisms [@doi:10.1038/s41467-018-03184-1; @doi:10.3389/fpls.2019.00846].
Here, we demonstrate that the taxonomic distance between the biological source of the queried compound and the biological source of the candidate structures is a valuable attribute to integrate into the metabolite annotation process.
We show that such information can be considered in a *taxonomically informed scoring* system and automatically applied to the outputs of different computational metabolite annotation programs.
The consideration of taxonomic information was shown to systematically improve the F1 score of the evaluated solutions ([ISDB](#isdb)-[DNP](#dnp), Sirius, MS-Finder) with a 1.5 to 7-fold increase.
The advantage of considering such information in the metabolite annotation process are thus observed *independently* of the tools and their associated structural [DB](#db)s.

It is worth noting that this benchmarking was carried to evaluate the importance of considering taxonomic information during the metabolite annotation process.
*It was not meant to compare the performances of the tools*.
Indeed, all compounds of the benchmarking dataset are present in the [DNP](#dnp), and the [ISDB](#isdb)-[DNP](#dnp) tool, which is by definition backed by the same [DB](#db) is thus favored.
On the other hand, the [GNPS](#gnps) spectral libraries were also part of the Sirius training set.
Furthermore filters for the selection of [M+H]^+^ adducts and for the filtering [MS/MS](#msms) spectra (500 most intense peaks) were applied to meet restriction of the [ISDB](#isdb)-[DNP](#dnp) and MS-Finder, respectively.
Finally, a number of entries (197) of the benchmarking dataset were found to have large mass difference (> 0.01 Da) between their experimental parent ion mass and their calculated exact mass. For example, cevadine [M+H]^+^ (CCMSLIB00004689734) had an experimental parent ion mass of 632.386 Da, while its calculated exact mass is 591.3407 Da (C~32~H~49~NO~9~).
Of course, such erroneous entries cannot be identified by the computational metabolite annotation tools (the list of these problematic entries is available online Problematic_entries.csv).
Altogether these elements prevent a fair comparison of each tool’s performances.
Another precautionary statement concerns the results of the optimization on candidate datasets for which taxonomic information had been randomly degraded at multiple taxa level.
This optimization indicated, for the [ISDB](#isdb)-[DNP](#dnp) results, that the optimal combination of scores was 0.81, 1.62 and 2.55 (family, genus and species taxa level, respectively).
Such results should be taken with caution, and not as absolute optimal values, as such optimization process are heavily dependent on the training sets.
Nevertheless, the optimization indicates that the best results were repeatedly obtained when the assigned scores were inversely proportional to the taxonomic distance between the biological sources of, both, the queried spectra and the candidate structures.

Other limitations of the described metabolite annotation strategy include its application range and prerequisites.
Indeed, it is important to note that such *taxonomically informed scoring* system will mostly benefit the annotation process of *specialized metabolites* and not ubiquitous molecules (e.g. coming from the primary metabolism) for obvious reasons.
Furthermore, it heavily depends on the availability and quality of [DB](#db)s compiling structures and their biological sources reported as a fully and homogeneously resolved taxonomy.
To the best of our knowledge, such [DB](#db)s are not publicly available and downloadable at the moment.
KNApSAcK ([http://www.knapsackfamily.com/KNApSAcK/](http://www.knapsackfamily.com/KNApSAcK/)) is a comprehensive species-metabolite relationship database compiling 116,315 metabolite-species pairs entries, it is accessible online but not downloadable.
Other databases such as FooDB ([http://foodb.ca](http://foodb.ca)) are fully downloadable but however focused on food-related metabolites, furthermore the biological sources field is not standardized.
The NPAtlas ([https://www.npatlas.org/](https://www.npatlas.org/)) is an interesting initiative, however biological sources information down to the species level is only accessible in query mode and the [DB](#db) is limited to 24,594 metabolites of microbial origin only.
The Dictionary of Natural Products, which we used in this study is the widest compilation of structure/biological sources pairs, but is only available commercially.
Furthermore, the biological sources are reported as a free text field (codes are available only for the family taxa levels and above), thus requiring tedious standardization and name resolving.

It is therefore important for the community to start the systematic reporting of biological sources, together with spectral and structural information, when documenting novel metabolites.
In fact, reporting newly described biological occurrence should be encouraged even for previously described metabolites.
However, the policy of most journals in [NP](#np) research is to accept for publication only description of novel and bioactive structures, which hinders these potentially informative reports.
The [GNPS](#gnps) spectral libraries ([https://gnps.ucsd.edu/ProteoSAFe/libraries.jsp](https://gnps.ucsd.edu/ProteoSAFe/libraries.jsp)) and MassIVE repositories ([https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp](https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp)) appear as optimal places, at the moment, to compile and share [NP](#np) spectral and structural information.
However, although free text comments can complement the documentation of an entry, no standardized fields are available to report the biological sources of the uploaded spectra.
The creation of such a feature, ideally directly linking the entered biological sources to existing taxonomy backbones such as GBIF ([https://www.gbif.org](https://www.gbif.org)) or Catalogue of Life ([http://www.catalogueoflife.org](http://www.catalogueoflife.org)), would be extremely useful.
A recent initiative, the Pharmacognosy Ontology (PHO), that builds on 50 years of development of NAPRALERT ([https://www.napralert.org](https://www.napralert.org)) is aimed at providing a Free and Open resource that will link taxonomical, chemical and biological data ([http://ceur-ws.org/Vol-1747/IP12_ICBO2016.pdf](http://ceur-ws.org/Vol-1747/IP12_ICBO2016.pdf)).
Of course, and in addition to correct and systematic biological sources occurrence reporting in dedicated [DB](#db)s, it is of utmost importance to count on the expert knowledge of trained taxonomists specialized in the classification of living organisms.
But it seems that today, unfortunately, these people are very few [@doi:10.1038/471037d; @doi:10.1525/bio.2011.61.12.4].

Building on the proposed taxonomically informed scoring, further developments will pass by the consideration of more accurate quantification of taxonomic distances and by strengthening the metascoring system.
Indeed, the approach presented here only considers the identity between the biological sources, at different taxa level, of the query compounds and the ones of the candidate structures.
Taking into account a more precise phylogenetic position within or across taxa, for example via the calculation of taxonomic distinctiveness indexes [@doi:10.1525/bio.2011.61.12.4; @doi:10.1111/j.1366-9516.2006.00234.x], could offer a more accurate distance and eventually improve such taxonomically informed scoring process.
Such calculations could however reveal to be computationally demanding to realize on the fly.
On another plan, efforts remain to be done towards the establishment of a global metascore (see Figure @fig:taxo-1).
Integrating the proposed taxonomic distance scoring (S~2~ in Figure @fig:taxo-1) with the maximal number of available metadata (S~3~, S~4~, …) when proceeding to metabolite annotation should only be beneficial to such process.
However, problematics such as the individual weights (w~1~, w~2~, w~3~,…) to attribute to each individual score of the metascore will have to be addressed.
