## Results and Discussion

This section is structured as follows: 
First, an overview of the process is illustrated.
The main technical elements are then detailed.
Then, the different results of the process are illustrated and the added value of the approach is demonstrated for a typical plant ethanolic extract of industrial interest, *Swertia chirayita* (Roxb.) H. Karst.
In addition, a comparison of the results with the literature is shown.
Some of the limitations of the method, related to peak shape and annotation are finally presented.

### Workflow

A scheme of the general workflow is presented in Figure @fig:cascade-1.

![**Schematic view of the presented workflow.** After the LC-MS/MS-CAD acquisition, the data are subjected to several processing steps. On the CAD side, a first pre-processing step is performed to enhance the signal. Then, peaks are automatically detected and integrated. On the MS/MS side, features are detected, grouped according to Ion Identity Networking and clustered into a Molecular Network. They are then annotated at the chemical class and/or structure level. A peak shape similarity calculation then links together the CAD peaks and MS features. An additional feature filtering step based on the knowledge of the sample can be performed. This provides combined qualitative and semi-quantitative information that can be explored in multiple ways.](images/cascade_workflow.pdf "cascade-1"){#fig:cascade-1 short-caption="Schematic View of the Presented Workflow" width="100%"}

As a minimal input, the workflow requires LC-MS/MS data, supplemented by another detector (here CAD and PDA, as described in [Data Acquisition](#data-acquisition)).
It then needs to be converted to an open format such as mzML, with all detectors encoded into the same file (see [Data Conversion](#data-conversion)).
Multiple processing steps then follow and will be detailed in the next subchapters.
First, signals are aligned as described in [Signals Alignment](#signals-alignment).
Being very noisy, the CAD required additional processing as described in [CAD Processing](#charged-aerosol-detector-cad-processing).
After several signal processing steps described below, a qualitative and semi-quantitative report is generated, as well as visual means to interrogate the data.
In parallel, MS features are extracted, clustered, and annotated, as described in [MS/MS Processing](#tandem-mass-spectrometry-msms-processing).
Afterward, MS features are attributed to CAD peaks as described in [Link Between CAD and MS Data](#link-between-cad-and-ms-data).
This allows signals to be categorized as belonging to 'major' or 'minor' metabolites.
Finally, to facilitate further investigation, different automated numerical and visual reports are generated as described in [Data Visualization](#data-visualization).

### Data Acquisition 

As previously mentioned, the minimal input of the workflow is LC-MS/MS data, supplemented by at least another detector.
Multiple trials were performed, with different columns lengths and gradient lengths (and flow), to try to achieve the best possible peak capacity (data not shown).
For the example illustrated in this chapter, UHPLC runs of 126 minutes on a 150mm column were performed.
This corresponded to a peak capacity of 480 at 400g/mol and 40Â°C (for comparison, a run of 7 minutes on a 50mm column corresponded to a peak capacity of 199, calculated with HPLC Calculator version 3.1) [@doi:10.1016/j.ejpb.2007.06.018].
UV spectra were acquired between 200 and 500nm and MS spectra between 100 and 1500*m/z*.
CAD scans were collected with a frequency of 100Hz, and a power function of 1.
The details of the data acquisition method are available in the section @sec:mm-da.

### Data Conversion 

All raw data files were converted using ThermoRawFileParser [v.1.4.0](https://github.com/compomics/ThermoRawFileParser/releases/tag/v1.4.0) [@doi:10.1021/acs.jproteome.9b00328].
Currently, only ThemoRawFileParser is able to encode the CAD signal directly in the mzML file from the raw file (with the `--allDetectors` option).
This feature was added by the authors of the tool on our request (see <https://github.com/compomics/ThermoRawFileParser/pull/87> and <https://github.com/compomics/ThermoRawFileParser/issues/60>). 

Once converted, the mzML file contained all requested signals, with their related chromatograms and spectra.
The chromatograms were encoded as `BasePeak_0` for the BPI chromatogram, `PDA.1_TotalAbsorbance_0` for the UV chromatogram, and `UV.1_CAD_1_0` for the CAD chromatogram.

More details about the data conversion method are available in the section @sec:mm-dc.

### Data Processing

#### Signals Alignment

The first necessary step was to align the signals of the different detectors. 
In the configuration used, the non-destructive PDA detector was placed first, before the split.
The MS signal was delayed by 0.090 and 0.055 minutes in comparison to the one of the PDA and the CAD, respectively.
These values were used to align the signals as shown in Figure @fig:cascade-2.

![**Alignment of the different signals after preprocessing of each signal.** All signals were preprocessed to improve the downstream steps. CAD and PDA signals are at the top, in red and green, respectively. MS signals (positive and negative ionization modes) are at the bottom, in light and dark blue, respectively. A zoomed-in area showing the different intensities normalized to the observed signal is also illustrated.](images/cascade_example_chromatogram.pdf "cascade-2"){#fig:cascade-2 short-caption="Alignment of the Different Signals After Preprocessing of Each Signal" width="100%"}

Figure @fig:cascade-2 highlights the complementarity between each signal obtained.
While the PDA intensity approximately followed the CAD intensity when a signal was present, some major molecules without a chromophore would have been ignored if PDA only is used.
The same complementarity could be observed between MS ionization modes (see light and dark blue traces).
While each CAD major peak had its corresponding peak in MS, the opposite did not hold true, as illustrated by the positive ionization mode at the end of the chromatogram.
Finally, although the presented approach also works with a single MS ionization mode, it is highly recommended to use both to improve coverage.

#### Charged Aerosol Detector (CAD) Data

As the CAD signal was very noisy, it required some preprocessing steps before peak extraction. 
These are briefly described hereafter and more in detail in the related Materials and Methods section.
First, a Fourier transform was performed, to keep only 1% of the components.
Fourier transform techniques to reduce noise were already demonstrated as powerful [@doi:10.1016/j.trac.2021.116354].
This already allowed to greatly lower signal interferences.
Moreover, the chromatographic resolution was improved using the derivative enhancement approach as described in [@doi:10.1016/j.talanta.2018.09.048].
This resulted in a sharper signal, where peak detection could be performed with better results than on the raw signal. 
To ensure the same processing between the different detectors, this was also done on the other signals.

Peaks were then integrated using simple algorithmic as CAD signal contains only one dimension, thus not allowing for more elaborated techniques, such as the ones used in MS and PDA peak detection.
A comparison of the peaks detected automatically before and after signal processing is shown below.

![**Comparison of peak detection on the raw and processed signal.** The results of the automated peak picking on the raw signal are shown on the top. Before processing, no peaks are detected due to the noise in the signal. The result of the peak detection and integration on the processed signal is shown at the bottom. Almost no peak is lost and only poorly-shaped peaks are split. A zoomed-in area showing more into detail the signal differences is also illustrated.](images/cascade-3.pdf "cascade-3"){#fig:cascade-3 short-caption="Comparison of Peak Detection on the Raw and Processed Signal" width="100%"}

Multiple iterations were performed in order to find parameters allowing at the same time to pick small peaks not splitting big ones (data not shown).
The result illustrated in Figure @fig:cascade-3 appeared as the best compromise.
While no peaks were automatically integrated before processing (top of the Figure), the majority of the visible peaks were correctly integrated (bottom of the Figure).
Very small, poorly defined peaks were not picked, while major peaks were.
This allowed for a good CAD peak-shape comparison with MS features.
Even if the manual integration of such data would perform better, automated pick peaking is still a challenge, as illustrated by diverse recent efforts trying to include machine learning to solve it [@doi:10.1021/acs.analchem.9b04811; @doi:10.1021/acs.analchem.1c02220].

#### High-Resolution Tandem Mass Spectrometry (HR-MS/MS) Data

##### Features Extraction 

MS features were extracted using MZmine3 v.3.1.0-beta [@doi:10.1186/1471-2105-11-395]. 
Trials were also made using Sirius 5.7.0 [@doi:10.1038/s41592-019-0344-8; @doi:10.1038/s41587-021-01045-9; @doi:10.1038/s41587-020-0740-8; @doi:10.1038/s42256-020-00259-x; @doi:10.1073/pnas.1509788112; @doi:10.1093/bioinformatics/btn603].
The main advantage of Sirius was that it did not require any parameter, thus making it optimal for automation.
As the peak picking algorithmic and adduct recognition was written by the authors of the tool themselves, downstream steps performed better.
Sirius performed well in the PI mode, but the NI mode led to inconsistencies thus MZmine3 was preferred for both modes (data not shown).
Peakonly [@doi:10.1021/acs.analchem.9b04811] was also used, but its integration with the rest of the workflow, needing finely defined isotope patterns, also led to worse performances.
Finally, some tests were performed using SLAW [@doi:10.1021/acs.analchem.1c02687], but they did not reach the performance of Sirius and MZmine3.
Integration of MZmine3 and IIN in SLAW in the future might improve a lot its performance and speed.

The MGF and the IIN edges exported from MZmine3 were used to build an MN through the GNPS platform ([fd78b0a89d6d40168fc43d5f13f71688](https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=fd78b0a89d6d40168fc43d5f13f71688) for positive and [033b5dcf471641c09907704696b82bb0](https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=033b5dcf471641c09907704696b82bb0) for negative).
The IIN was complemented using the MS1 adduct module of TIMA v2.5.5 [@doi:10.3389/fpls.2019.01329; @doi:10.5281/zenodo.6929297].
Thus, adducts, multimers, and fragments generated by the same molecule could be annotated as such, even without structural annotation.

##### Features Annotation 

MS features were annotated using a multi-tool approach.
They were annotated using Sirius 5.7.0 and complemented with annotations coming from the GNPS libraries, together with ISDB-LOTUS.
Such an approach allows using the strengths of each tool while reducing weaknesses thanks to the other ones.
The combination of the annotations of all three tools was then submitted to taxonomically informed metabolite annotation using TIMA v2.5.5 [@doi:10.3389/fpls.2019.01329; @doi:10.5281/zenodo.6929297].
This way, multi-charged molecules, for example, could be annotated.
After this, only the best candidate (based on the final score after *taxonomically informed scoring*) (see Figure @fig:taxo-1) was kept for later steps.
Further, only confident best candidates were kept.
All details are given in the corresponding Materials and Methods section.

In positive mode, this led to XXX `//TODO`{.red} annotations, of which XXX `//TODO`{.red} were confident (XX,X%) on 3,610 features.
In negative mode, this led to XXX `//TODO`{.red} annotations, of which XXX `//TODO`{.red} were confident (XX,X%) annotations on 2,312 features.

#### Link Between CAD and MS Data

Independently from features annotation, CAD and MS peaks shapes were compared.
As CAD peaks could contain multiple features, features with the best peak-shape correlation with the CAD peak were also the most legitimate ones to account for the CAD signal.
As a first mandatory step, all points of CAD and MS peaks were inter- and extrapolated to the same frequency, to be defined with approximately the same number of points.
This is generally not needed as peak shapes usually compared, are coming from the same detector.
This usually results in the same frequency of acquisition. Here, since the frequency of the CAD is approximately 30 times higher than the one of MS, resampling was performed.

The second step consisted of normalizing the peakâs intensity together with a retention time normalization between the two peakâs minima (illustrated in green in Figure @fig:cascade-3).
The aim of this procedure was to improve peak shape similarity between the compounds whose ionization strongly differs from the CAD trace.
Without this step, the MS features with the highest intensity would by default have a more defined peak shape and thus be more likely to be correlated to the CAD signal.

Only features with a similarity â¥ 0.8 were considered, with the possibility to adapt the threshold.

If an MS feature had a correlation â¥ 0.8, its corresponding annotation was considered as a "major" compound, else, it was categorized as a "minor" compound.
This allowed distinguishing major and minor features even if they were all found below the same CAD peak.
All features outside of a CAD peak were considered âminorâ by default.
The higher the peak capacity [@doi:10.1021/ac50157a075], the better the classification.
Future applications, such as coreâshell or sub-micrometer porous particles packed columns [@doi:10.1016/j.chroma.2014.11.069], are expected to greatly help in this regard.

CAD peaks could envelop multiple or only a few (non-)peak shape correlated features.
An example of a CAD peak containing multiple MS-features in both PI and NI modes is illustrated in Figure @fig:cascade-4.

![**Link Between MS-features and CAD Peak.** Each panel is composed of positive and negative ionization modes (negative ionization mode shown as negative values). The MS features are represented with plain lines and CAD peak with dashed line. On panel A, features are colored according to their peak shape similarity with the CAD peak. On panel B, features are colored according to the molecular formula of the best candidate. On panel C, features are colored according to the 2D structure of the best candidate. On panel D, features are colored according to the `taxonomic distance score` part of the *taxonomically informed scoring* (a value close to one indicates the 2D structure has already been reported in a similar organism).](images/cascade-4.pdf "cascade-4"){#fig:cascade-4 short-caption="Link Between MS-features and CAD Peak" align="center" width="100%"}

Each panel is composed of PI and NI modes (NI mode shown as negative values). 
The MS features are represented with plain lines and CAD peak with dashed line.

First, on panel A, features are colored according to their peak shape similarity with the CAD peak. 
In the example, peaks with a higher intensity also share a higher similarity.
This is mainly due to the fact that because of their intensity, their peak shape is initially better defined.
The resampling step described before helped to limit the effect of intensity on peak shape similarity calculation.
Nevertheless, MS-features with low intensity are less likely share a good peak shape similarity with the CAD peak. 

On panel B, features are colored according to the molecular formula of the best candidate.
Formulas occurring less than two times were grouped in the "other" category.
A good consistency between the annotations in positive and negative modes can be observed, with C~30~H~48~O~3~ being the most consistent formula.
The features annotated consistently also share good peak shape similarity on panel A. 

On panel C, features are colored according to the 2D structure of the best candidate.
Structures occurring less than two times were grouped in the "other" category.
This further refines panel B, showing that some of the features annotated with the same molecular formula, C~30~H~48~O~3~, do not share the same structural annotation (MIJYXULNPSFWEK and WCGUUGGRBIKTOS, both triterpenoid derivatives, corresponding to the 2D structures of [oleanolic acid](https://www.wikidata.org/wiki/Q418628) and [ursolic acid](https://www.wikidata.org/wiki/Q416260), respectively).

On panel D, features are colored according to the `taxonomic distance score` part of the *taxonomically informed scoring*. 
A value close to one indicates the 2D structure has already been reported in a similar organism than the one studied (here, *S. chirayita*).
Both MIJYXULNPSFWEK and WCGUUGGRBIKTOS corresponded to 2D structures already reported in *S. chirayita* [@doi:10/bqwnm4].

All together, the different relations between these features and their respective CAD peak could help gaining information while reducing the complexity of the data.
Informed filtering of the features considered as belonging to the major metabolome is detailed in the next subchapter.

#### Major Compounds Filtering 

Having all features informed as previously described, guided data reduction could be performed.
This step was necessary to further narrow down the amount of information to be finally reported and analyzed.
Basic statistics illustrating the number of features for each ionization mode, together with their link to their attributes are given in Table @tbl:cascade-1.

Table: Initial MS features - CAD peaks statistics {#tbl:cascade-1}

|                                 |       **Positive**       |        **Negative**      |
|:------------------------------- | ------------------------:| ------------------------:|
| Total features                  | 3,610                    | 2,312                    | 
| Total CAD peaks                 | 44                       | 38                       | 
| Features linked to CAD peak     | 714                      | 489                      |
| Features per peak               | 16.2                     | 12.9                     | 
| Structures per peak             | 14.3                     | 10.8                     | 
| Molecular formulas per peak     | 14.3                     | 10.8                     | 
| Chemical classes per peak       | 11.3                     | 9.1                      | 

Overall, more features were detected in PI, compare to NI (3,610 vs 2,312).
Not only the total features were higher in PI, but also the number of features per CAD peak.
This is in line with previous conclusions stating that, while PI mode is usually preferred as it is expected to be more generic, the ionization efficiency (ratio to noise) is better in NI mode [@doi:10.1021/acs.analchem.7b00096].
A small difference of CAD detected peaks can be observed between the two acquisition. 
Polarity switching experiments with MS instruments reaching greater acquisition frequency, thus allowing to acquire PI and NI modes in a single run could avoid this difference.

These metrics can also be used to evaluate the peak purity [@doi:10/dx5qvh; @doi:10/bjmvb9].
The concept of peak purity could then be used to evaluate the quality of the acquired fragmentation spectrum [@doi:10.1021/acs.analchem.6b04358], or even try to deconvolute it [@doi:10.1021/ac400751j].

As the presented workflow aims at linking MS features to CAD peaks to categorize them as "major" or "minor" metabolites, the only fact that the features were under a CAD envelope was not sufficient. 
Filtering strategies were therefore developed to narrow down the number of features considered as "major".
The first step consisted in applying the peak shape similarity described before.
A comparison of the features statistics before and after filtering is made in Table @tbl:cascade-2.
The corresponding Table for NI mode is available in Appendix @sec:appendix-cascade-1.

Table: Peak shape similarity filtered features statistics (positive ionization mode) {#tbl:cascade-2}

|                                 |   Before similarity filter  |   After similarity filter   |
|:------------------------------- | ---------------------------:| ---------------------------:|
| Features linked to CAD peak     | 714                         | 284                         |
| Features per peak               | 16.2                        | 7.5                         | 
| Structures per peak             | 14.3                        | 6.6                         | 
| Molecular formulas per peak     | 14.3                        | 6.6                         | 
| Chemical classes per peak       | 11.3                        | 5.8                         | 

As this filtering process was effective but not sufficient (over five structures/formulas per peak).
Therefore, an additional filter was implemented to further narrow down the number of features attributed to the major compounds.
This filter consisted in favoring compounds already known in the genus of the studied extract.
If the compound was already isolated from the same source, the probability of it being a major compound is higher. 
If a compound was found in the studied species, all other "major" candidate compounds were switched to the "minor" category, except if they were also found in the species.
The same rule was then applied to the genus, giving priority to the compounds of the species.
This led to fewer features considered "major", with less than two features remaining per CAD peak, as described in Table @tbl:cascade-3.
The corresponding Table for NI mode is available in Appendix @sec:appendix-cascade-2. 

Table: Peak shape similarity and taxonomy filtered confident features statistics (positive ionization mode) {#tbl:cascade-3}

|                                 | After similarity filter only|After both filters (similarity+taxonomy) |
|:------------------------------- | ---------------------------:| ---------------------------:|
| Features linked to CAD peak     | 284                         | 66                          |
| Features per peak               | 7.5                         | 2.1                         | 
| Structures per peak             | 6.6                         | 1.7                         | 
| Molecular formulas per peak     | 6.6                         | 1.7                         | 
| Chemical classes per peak       | 5.8                         | 1.6                         | 

After the complete filtering process, in average, each CAD peak apppears to have an almost unambiguously attributed structure.
The number of features and structures that could be attributed to a CAD peak was lowered by a factor of almost 10 (2.1 and 1.7 against 16.2 and 14.3 initially).
In fact, even if there are annotated features under their envelope at the beginning, some of the CAD peaks are not actually linked to any confidently annotated and peak-shape correlated feature after the filtering.
Only CAD peaks with linked features were therefore taken into account into  the calculations.

Therefore, to complement the averages showed in Tables @tbl:cascade-1, @tbl:cascade-2, and @tbl:cascade-3, an alluvial plot summarizing the filtering process on the PI data is illustrated in @fig:cascade-5.

![**Alluvial Plot of the Informed Data Filtering Process.** The underlying data is PI mode. Each alluvial plot is colored according to five categories. The categories correspond to the number of features per CAD peak in panel A and to the number of structures per CAD peak in panel B. The first block (from left to right) represents the situation before filtering, the second block after peak-shape filtering, and the last block after peak shape filtering, taxonomical filter and annotation confidence filter.](images/cascade-5.pdf "cascade-5"){#fig:cascade-5 short-caption="Alluvial Plot of the Informed Data Filtering Process" align="center" width="100%"}

This stringent filtering allowed to avoid part of the noise generated by automated annotation into account, and focus on consistent metaboites only.
The filtering did not only reduce the number of candidates linked to a CAD peak, but also discarded the ones whose annotation was not confident.
Starting from a situation where no CAD peak was unambiguously linked to a single structure, almost half of the CAD peaks can be unambiguously linked to a structure in the end. 
Thus, the unambiguous annotation of the major metabolome could be automatized and peaks where some ambiguity remains were reduced.

### Data Visualization

To facilitate further exploitation of the data, several visualizations of the processed data are generated.
First, pseudochromatograms reflecting the biological origin of the annotated compounds with respect to the biological source of the extract are presented in Figure @fig:cascade-6.

![**Taxonomically Informed Pseudochromatograms.** The bars are colored according to the taxon level of the nearest organism the compound was find in in respect to the extract biological source. Dark green colored bars mean the compound was already reported in the species. The remaining categories follow the same logic. Panel A illustrates the annotations in positive ionization mode while panel B in negative ionization mode.](images/cascade-6.pdf "cascade-6"){#fig:cascade-6 short-caption="Taxonomically Informed Pseudochromatograms" align="center" width="90%"}

In both panels, major CAD peaks are linked to green or dark green categories.
This further supports the logic of favoring compounds already isolated from near organisms as being attributed to CAD peaks.
While the majority of the CAD peaks are annotated in both modes, an interesting complementarity can be observed.

If *S. chirayita* is not an well studied species, nor the *Swertia* genus, the *Gentianaceae* family is (see Figure @fig:intro-1).
Therefore, its analysis can benefit from previously acquired data, as illustrated in previous chapters.
In the case of less studied organisms, belonging to a poorly studied branch of the tree of life, it will be necessary to rely more on the chemical classes and the condifence of the spectral score.

The colors of the bars can also be tailored to reflect the chemical pathways present in the extract [@doi:10.1021/ACS.JNATPROD.1C00399]. 
Then, these bars can be aggregated together as shown in Figure @fig:cascade-7.

![**Chemically Informed Pseudochromatogram and Related Treemaps.** On panel A and B, the bars are colored according to the chemical superclass attributed to the CAD peak. The data used was the positive ionization mode and intensities were normalized. On panel C and B, the bars are regrouped into categoories, independently from their position on the chromatogram. For the panel D, MS intensity was neglected and only the count of the bars is illustrated.](images/cascade-7.pdf "cascade-7"){#fig:cascade-7 short-caption="Chemically Informed Pseudochromatogram and Related Treemaps" align="center" width="100%"}

The majority of the CAD peaks of the extract could be attributed to the terpenoid pathway.
They accounted for 52% of the total area (see panel C).
Terpenoids also represented the majority of the "minor" signals.
The second most important pathway were the shikimates and phenylpropanoids (24%).
Then, a fair amount of peaks were attributed to the polyketide pathway.
This was because of the ambiguous biosynthetic origin of xanthones.
In plants, xanthones are known to be produced by the shikimate pathway [@doi:10.3389/fpls.2022.809497], but NPClassifier classified part of them as ambiguously coming from the polyketide pathway (mainly used in Fungi, see Figure @fig:magicTree) also.
Fungal endophytes are known to account for many of the NPs found in plants, and also xanthones [@doi:10.1080/07388551.2021.1940087].

More globally, the minor composition roughly followed the major one, with the exception of fatty acids and alkaloids, being overrepresented in the minor metabolome.
The unproportional amount of alkaloids detected can be attributed to their good ionization in PI.
Concerning the fatty acids, they constitue the foundatin of membranes. Even if the extraction process (see Materials and Methods) aimed at removing them, complete removal is difficult.
Moreover, they are essential building blocks for more evolved structures and observing them as a leftover of plants combinatorial bisynthesis is expected [@doi:10.1039/C1NP00049G].

#### Automated Reporting

An automated tabular report with the most relevant information can also be generated.
An example of such table is presented in Table @tbl:cascade-4.

\newpage

\tiny

Table: Automated Report of the Major Metabolome of *Swertia chirayita* {#tbl:cascade-4}

|Peak ID|Peak RT [min]|Peak Area [%]|Feature ID|Feature RT [min]|Feature *m/z*|Structure|InChIKey 2D|Annotation Score|Chemical Pathway|Chemical Superclass|Chemical Class|Nearest Organism\footnotemark{}|Reference(s)|
|-:|-:|-:|-:|-:|-:|-----|------|--:|----|----|-----|--|-|
|1|6.25|0.02|76|6.28|397.1109|image|HEYZWPRKKUGDCR|0.86|Terpenoids|Monoterpenoids|Secoiridoid monoterpenoids|Swertia chirayita|[@doi:10.1248/CPB.53.674; @doi:10.1002/PTR.2738; @doi:10.1016/J.PHYMED.2008.09.010; @doi:10.1248/CPB.18.1856]|
|2|7.05|0.04|94|7.09|423.0927|image|AEDDIBAIWPIIBD|0.85|Shikimates & Phenylpropanoids|Xanthones|Plant xanthones|Swertia|[@doi:10.1021/NP100008R]|
|3|19.34|0.02||||||||||||
|4|19.53|0.02|359|19.56|1205.3304|image|UZYZCCWBBBCDAD|0.77|Terpenoids|Monoterpenoids|Secoiridoid monoterpenoids|Swertia chirayita|[@doi:10.1021/NP960018G; @doi:10.1002/PTR.2738]|
|5|20.53|0.02||||||||||||
|6|20.75|0.03|376|20.79|1173.3404|image|DBOVHQOUSDWAPQ|0.60|Terpenoids|Monoterpenoids|Secoiridoid monoterpenoids|Swertia chirayita|[@doi:10/b9fmmg; @doi:10.1021/NP960018G; @doi:10.1002/PTR.2738]|
|7|21.4|0.02|384|21.38|465.1388|image|MSOFPKZBJVHCQU|0.70|Shikimates & Phenylpropanoids|Xanthones|Plant xanthones|Swertia|[@doi:10.3987/COM-11-12257]|
|8|21.92|0.02|440|21.96|417.1205|image|AEDDIBAIWPIIBD|0.64|Shikimates & Phenylpropanoids|Xanthones|Plant xanthones|Swertia|[@doi:10.1021/NP100008R]|
|9|23.08|0.01|458|23.11|683.1276|image|DVMVOSIIRBPMIY|0.70|Shikimates & Phenylpropanoids|Xanthones|Plant xanthones|Swertia||
|10|23.74|0.02|419|23.78|270.0885|image|OAIHXYJZSSASMS|0.53|Shikimates & Phenylpropanoids|Flavonoids|Flavones|Swertia||
|11|25.46|0.02|501|25.51|519.0582|image|MPXAWSABMVLIBU|0.77|Polyketides|Xanthones|Methyl xanthones|Swertia chirayita|[@doi:10.1002/JPS.2600620614; @doi:10.1080/14786410600906319]|
|12|26.17|0.02|514|26.21|1387.2273|image|GNMLPEJAIRBAAO|0.59|Shikimates & Phenylpropanoids|Xanthones|Plant xanthones|Swertia|[@doi:10.1021/NP100008R; @doi:10/d9s28w; @doi:10.3987/COM-11-12257; @doi:10.1021/JO00025A032]|
|13|27.27|0.01|516|27.35|455.0967|image|KMOUJOKENFFTPU|0.59|Shikimates & Phenylpropanoids|Flavonoids|Flavones|Swertia|[@doi:10.1055/S-2007-971248]|
|14|27.92|0.02||||||||||||
|15|28.6|0.02||||||||||||
|16|29.23|0.03||||||||||||
|17|29.38|0.04||||||||||||
|18|29.51|0.02|586|29.56|519.0583|image|MPXAWSABMVLIBU|0.60|Polyketides|Xanthones|Methyl xanthones|Swertia chirayita|[@doi:10.1002/JPS.2600620614; @doi:10.1080/14786410600906319]|
|19|29.73|0.04||||||||||||
|20|30.34|0.02||||||||||||
|21|36.45|0.01|775|36.47|455.0636|image|KIJTTYRRBNSKMV|0.53|Polyketides|Xanthones|Methyl xanthones|Swertia||
|22|37.45|0.02|797|37.45|287.0567|image|DFDHXNQBZDTVFE|0.60|Polyketides|Xanthones|Methyl xanthones|Swertia chirayita|[@doi:10.1248/YAKUSHI1947.89.3_410]|
|23|38.61|0.02|824|38.64|423.1094|image|AJENETGXVIBLQY|0.70|Terpenoids|Monoterpenoids|Secoiridoid monoterpenoids|Swertia|[@doi:10.1002/HLCA.200890134]|
|24|38.82|0.02|830|38.88|273.0410|image|BDBVOZGRVBXANN|0.60|Polyketides|Xanthones|Methyl xanthones|Swertia chirayita|[@doi:10.1080/14786419.2011.592836; @doi:10.1016/J.JEP.2013.12.032; @doi:10.1155/2014/542385]|
|25|39.84|0.01|861|39.9|341.0284|image|BDBVOZGRVBXANN|0.60|Polyketides|Xanthones|Methyl xanthones|Swertia chirayita|[@doi:10.1080/14786419.2011.592836; @doi:10.1016/J.JEP.2013.12.032; @doi:10.1155/2014/542385]|
|26|44.67|0.02|1034|44.75|303.0862|image|VYRIGRQQKUZPEX|0.60|Polyketides|Xanthones|Methyl xanthones|Swertia chirayita|[@doi:10.1002/JPS.2600620614]|
|27|46.35|0.02|1056|46.39|531.0584|image|SWDVHXLKYHFUOZ|0.70|Polyketides|Xanthones|Methyl xanthones|Swertia||
|28|49.22|0.01|1175|49.3|289.0714|image|PUECEVJMPDNNHT|0.89|Polyketides|Xanthones|Methyl xanthones|Swertia chirayita|[@doi:10.1002/JPS.2600620614]|
|29|49.96|0.03|1202|50.05|289.0712|image|DFDHXNQBZDTVFE|0.60|Polyketides|Xanthones|Methyl xanthones|Swertia chirayita|[@doi:10.1248/YAKUSHI1947.89.3_410]|
|30|52.23|0.02|1274|52.38|437.3420|image|KUFXJZXMWHNCEH|0.53|Terpenoids|Sesquiterpenoids|Eudesmane sesquiterpenoids|Swertia|[@doi:10.1246/BCSJ.56.3477]|
|31|58.4|0.02|1469|58.36|473.3640|image|KLHSKTMVSOWVLD|0.64|Terpenoids|Triterpenoids|Oleanane triterpenoids|Gentianales|[@doi:10.1248/CPB.32.3906; @doi:10/bft85t]|
|32|59.85|0.02|1481|59.97|440.1508|image|WHUUTDBJXJRKMK|0.53|Amino acids & Peptides|Small peptides|Aminoacids|Swertia|[@doi:10.1055/S-0028-1097323]|
|33|66.4|0.01|1702|66.24|232.1464|image|GZCGUPFRVQAUEE|0.53|Carbohydrates|Saccharides|Monosaccharides|Swertia|[@doi:10.4268/CJCMM20151921]|
|34|75.39|0.16|2004|75.53|913.7343|image|WCGUUGGRBIKTOS|0.77|Terpenoids|Triterpenoids|Ursane & Taraxastane triterpenoids|Swertia chirayita|[@doi:10/bqwnm4]|
|35|79.41|0.01|2109|79.52|425.3793|image|OMUHPIZFDFBYLO|0.71|Terpenoids|Triterpenoids|Taraxerane triterpenoids|Swertia chirayita||
|36|81.42|0.02||||||||||||
|37|86.88|0.03|2061|86.96|255.2334|image|VSMOENVRRABVKN|0.53|Fatty acids|Fatty acyls|Fatty alcohols|Swertia|[@doi:10.1246/BCSJ.56.3477]|
|38|88.62|0.01|2098|88.74|471.3498|image|CJRASULCEXTLGY|0.57|Terpenoids|Triterpenoids|Ursane & Taraxastane triterpenoids|Gentianaceae||
|39|92|0.02|2476|91.87|425.3797|image|NINOWLHIYZJIFS|0.71|Terpenoids|Triterpenoids|Multiflorane triterpenoids|Swertia chirayita|[@doi:10/bqwnm4; @doi:10.1039/C39890000438]|
|40|97.88|0.01|2649|97.6|611.1687|image|PNRKIXSZTUCIJN|0.59|Shikimates & Phenylpropanoids|Phenylethanoids (C6-C2)|Phenylethanoids|Swertia|[@doi:10.1248/CPB.53.48]|
|41|101.19|0.02|2760|101.33|596.4204|image|NPJICTMALKLTFW|0.53|Terpenoids|Steroids|Stigmastane steroids|Swertia|[@doi:10.1016/J.CCLET.2009.02.006]|
|42|105.2|0.01||||||||||||
|43|115.58|0.02|3203|115.64|355.0699|image|MIPXRVRWIMVNPA|0.77|Shikimates & Phenylpropanoids|Xanthones|Plant xanthones|Swertia chirayita|[@doi:10.1007/S10870-005-2802-7]|
|44|118.1|0.02||||||||||||

\footnotetext{Organism where the structure was already repoorted with the lowest taxonomic distance}

\normalsize

Of the 44 CAD peaks, 15 were associated with 2D structures previously reported in *S. chirayita*, and 31 in the genus *Swertia*.
One peak was associated with a 2D structure found in *Gentianaceae* and one in *Gentianales*. 
11 CAD peaks remained without confident and unambiguous annotation.
These peaks should be prioritized for further study.
The majority of the annotated structures were xanthones.
For the convenience of the analyst or to suggest specifications for a group of compounds, the @tbl:cascade-4 table can be aggregated by compound classes.
The two tables correspond to the two views shown in Figure @fig:cascade-7, but this time with PI and NI merged.
In this way, the composition of the extract can be approached more holistically, with compounds sharing similar properties (here, structural) grouped together.
An condensed report is shown in Table @tbl:cascade-5.

\tiny

Table: Automated Condensed Report of the Major Metabolome of *Swertia chirayita* {#tbl:cascade-5}

| **Group**| **Peak ID**| **Peak Area [%]**| **Group**| **Peak ID**| **Peak Area [%]**|
|-------------------------|----------|----------|-------------------------|----------|----------|
| **Unknown**| 3| 0.02| **Secoiridoid monoterpenoids**| 1| 0.02|
|| 5| 0.02|| 4| 0.02|
|| 14| 0.02|| 6| 0.03|
|| 15| 0.02|| 23| 0.02|
|| 16| 0.03| Total|| **0.09**|
|| 17| 0.04| **Flavones**| 10| 0.02|
|| 19| 0.04|| 13| 0.01|
|| 20| 0.02| Total|| **0.03**|
|| 36| 0.02| **Fatty alcohols**| 37| 0.03|
|| 42| 0.01| Total|| **0.03**|
|| 44| 0.02| **Eudesmane sesquiterpenoids**| 30| 0.02|
| Total|| **0.26**| Total|| **0.02**|
| **Methyl xanthones**| 11| 0.02| **Oleanane triterpenoids**| 31| 0.02|
|| 18| 0.02| Total|| **0.02**|
|| 21| 0.01| **Aminoacids**| 32| 0.02|
|| 22| 0.02| Total|| **0.02**|
|| 24| 0.02| **Multiflorane triterpenoids**| 39| 0.02|
|| 25| 0.01| Total|| **0.02**|
|| 26| 0.02| **Stigmastane steroids**| 41| 0.02|
|| 27| 0.02| Total|| **0.02**|
|| 28| 0.01| **Monosaccharides**| 33| 0.01|
|| 29| 0.03| Total|| **0.01**|
| Total|| **0.18**| **Taraxerane triterpenoids**| 35| 0.01|
| **Ursane and Taraxastane triterpenoids**| 34| 0.16| Total|| **0.01**|
|| 38| 0.01| **Phenylethanoids**| 40| 0.01|
| Total|| **0.17**| Total|| **0.01**|
| **Plant xanthones**| 2| 0.04||||
|| 7| 0.02||||
|| 8| 0.02||||
|| 9| 0.01||||
|| 12| 0.02||||
|| 43| 0.02||||
| Total|| **0.13**||||

\normalsize

While the condensed report presented here above is grouping compounds by chemical classes, the same methodology could be followed with other grouping variables, such as biollogical properties.

#### Literature Matching

As an approximation of compounds present in the extracts, a query of reported compounds in the *Swertia* genus was performed on Wikidata, which is now the largest occurrences resource for natural products [@doi:10.7554/eLife.70780].
The returned list of compounds for each producing organism was then filtered to better match the LC-MS run conditions (see Material and Methods). 

A comparison between the automated compositional assesment obtained by LC-MS and the one reported in the literature is illustrated in Figure @fig:cascade-8.

![**Comparison of the Experimental Data for *S. chirayita* With Literature Data.** Panel A illustrates the composition of *S. chirayita* according to the CAD informed LC-MS analyses in both positive and negative ionization modes. Panel B illustrates the composition of the *Swertia* genus according to the number of compounds reported in the literature [@doi:10.7554/eLife.70780] for each chemical class.](images/cascade-8.pdf "cascade-8"){#fig:cascade-8 short-caption="Comparison of the Experimental Data for *S. chirayita* With Literature Data" align="center" width="100%"}

Such a comparison provides a quick way to see if the analyses are consistent with what is reported in the literature [@doi:10.7554/eLife.70780].
Extraction methods can lead to selective favoring of certain classes of compounds and must be taken into account.
This is illustrated by the overrepresentation of triterpenoids in the experimental treemap (panel A of Figure @fig:cascade-8).
The studied ethanolic extract of *S. chirayita* underwent a concentration step to remove polar compounds, thus favoring more apolar compounds such as triterpenoids (see Materials and Methods).
In addition, the annotated triterpenoids correspond to the red peaks in Figure @fig:cascade-2.
This further reinforces the reason for using a CAD detector, as the PDA detector would not have detected these compounds, which represent the majority of the mass of the extract studied.

#### CAD-Informed Molecular Network (MN)

As the presented workflow not only allows to annotate peaks associated to the major metabolome, but also minor ones, MN appears as an efficient way to search for minor analogues present in the extract.

![**CAD-Informed Molecular Network.** Each node represents a feature. Features are colored according to the chemical pathway of their annotation. The size of the nodes corresponds to the peak area of their corresponding CAD peak. The type of edge connecting the nodes corresponds to the interaction the nodes share. Big and medium-sized nodes can serve as anchor points for further exploration. Only the biggest cluster (in positive ionization mode) is illustrated. The corresponding GNPS is [fd78b0a89d6d40168fc43d5f13f71688](https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=fd78b0a89d6d40168fc43d5f13f71688).](images/cascade-9.pdf "cascade-9"){#fig:cascade-9 short-caption="CAD-Informed Molecular Network" align="center" width="100%"}

`//TODO`{.red}

