# General Discussion {.page_break_before}

The objective of the presented developments is their integration in a more integrated chemical composition approach, from the plant (or the holobiont) to the molecule, and back.

`//TODO`{.red}

<!-- Interaction human microbiome non linear SECOM [@doi:10.1038/s41467-022-32243-x]

bio similaire chimie ppas similaire ubiquitaire + geo

bien insister données publiques computationnallles permet ces vues inaccessibles

balbutiements de l'automated compositional asessment

peut-être refaire parallèle genomique/proteomique -->

In reality, accessing *the metabolome* is still far away, letting the door open for further research.
This starts with better reporting practices for the occurrence of chemical compounds in living organisms.
As illustrated in Figure @fig:intro-1, at the scale of the tree of Life, the chemical knowledge is still very limited (5 compounds per organism, on the few organisms studied, Figure @fig:distribution).
Fortunately, research is progressing on model organisms that can serve as a basis for a more complete study of the rest.

To illustrate this idea, a Tanglegram of the 50 most chemically studied (not neccesarily model) organisms together with 50 randomly selected organisms is illustrated in Figure @fig:disc-tanglegram.

![**Tanglegram of the 50 most chemically studied and 50 randomly selected organisms in LOTUS.** On the left, organisms are clustered according to the classical, known, biological taxonomy. On the right, they are clustered based on their chemical content. The entanglement of both trees is shown, with conserved positions in tips colored. If the organism belongs to the most 50 chemically studied organisms, its label is black, else it is grey. The chemical matrix used to build the chemical tree was informed biosynthetically [@doi:10.1007/s00049-017-0250-4].](images/tanglegram_lotus_top50.pdf "disc-tanglegram"){#fig:disc-tanglegram short-caption="Tanglegram of the 50 Most Chemically Studied and 50 Randomly Selected Organisms in Lotus" align="center" width="100%"}

Figure @fig:disc-tanglegram highlights the close connection between biology and chemistry.
The randomly selected organisms (in grey), cluster relatively well with their biologically closest chemically studied organisms (in black).
The chemical matrix used to build the chemical tree was informed biosynthetically [@doi:10.1007/s00049-017-0250-4], and such biologically and chemically informed matrices [@doi:10.1038/s41589-020-00677-3; @doi:10.1038/ismej.2010.133] should take more place in metabolomics studies.

<!--                         Similarity to *Homo sapiens*
Homo sapiens                       1724.7861
Arabidopsis thaliana                645.9382
Escherichia coli                    583.5809
Trypanosoma brucei                  281.7078
Vitis vinifera                      205.6855
Drosophila melanogaster             176.4759 -->

The interesting clustering of model organisms in the chemical tree suggests that, when heavily studied, access to a common core metabolism is reached.
Once this common core metabolism will be better defined, the path to a better definition of specialized metabolism, novel metabolites [@doi:[@doi:10.1038/s41592-022-01486-3]], and their true biological origin will be open.

As for other fields, it is highly challenging and unconventional to generate *absence* data in metabolomics.
A first step will therefore be to model pseudoabsence [@doi:10/bctwxw; @doi:10.1371/journal.pone.0044486], to then be abe to better unravel what is present.
Then, future studies aiming at the prediction of Life's metabolism with modern computational tools will be possible.
Novel tools currenty developed to better automate [MS](#ms)-based search (such as MaSSQL [@doi:10.1101/2022.08.06.503000]) and accessible chemical ontologies [@doi:10.1515/pac-2021-2007] will be key players in this quest.
Better ways to express uncertainty in chemical annotation from spectral data will be needed. 
CxSMILES\footnotemark{} have a good chance of being successful in this regard.
\footnotetext{See \url{https://egonw.github.io/cdk-cxsmiles/}}

The purpose of the present work has been, in the most humble measure, to contribute to the enhancement of the chemical wealth of Nature.

![**Current knowledge of the chemistry of Life.** The tree generated from current LOTUS data builds on biological taxonomy and employs the biological kingdom as tips color. The bars correspond to the chemical classes found in the biological family.](images/tree_full.pdf "disc-full"){#fig:disc-full short-caption="Current Knowledge of the Chemistry of Life" align="center" width="100%"}